textvalue-compose/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+    convention.library.android
+}
+
+description = "Compose extensions for TextValue"
+
+dependencies {
+    api(projects.textvalue)
+    api(libs.androidx.compose.ui)
+}
+
+android {
+    namespace = "com.redmadrobot.textvalue.compose"
```

Comment:
Minor
```suggestion
    namespace = "$group.compose"
```

Reply:
For understanding: so everything we declare in gradle.props works as environment variables?

Reply:
Not sure about everything, but `group` and `version` - yes